ctapipe/irf/binning.py

```diff
@@ -0,0 +1,176 @@
+import astropy.units as u
+import numpy as np
+from pyirf.binning import create_bins_per_decade
+
+from ..core import Component
+from ..core.traits import Float, Integer
+
+
+class OutputEnergyBinning(Component):
```

Comment:
I'd better like this to be closer to where the things are actually used and split up into components.

We might not want to use the same energy binning for effective area and higher dimensional IRFs for example.